index.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300&display=swap" rel="stylesheet"> 
+    <script src="https://kit.fontawesome.com/7bc3c221d9.js" crossorigin="anonymous"></script>
+    <title>Practice</title>
+</head>
+<body>
+                    <!--NAVBAR-->
+<div class="navbar">
+    <div class="initial">
+    <img src="images/line.png" alt="line" >
+    <img src="images/YouTube.png" alt="line">
+</div>
+<div class="search">
+        <input type="text"  placeholder="Search.." > 
+    <div class="searchbtn">
+        <i class="fas fa-search"></i>
+    </div>
+
+</div>
+
+<div class="end">
+    <img src="images/camera.png" alt="line">
+    <img src="images/grid.png" alt="line">
+    <img src="images/bell.png" alt="line">
+</div>
+</div>
+
+                    <!--VIDEO PLAYER-->
+<div class="container"> 
```

Comment:
Use semantic HTML: here you should look for something more semantic than a `<div>`, you can check [this](https://www.w3schools.com/html/html5_semantic_elements.asp) link.